sys/fatfs_diskio/Makefile

```diff
@@ -0,0 +1,2 @@
+MODULE = fatfs_diskio
+include $(RIOTBASE)/Makefile.base
```

Comment:
If this module is fatfs specific: why are they in `sys` and not `pkg/fatfs/`? Please move them (sorry for ringing in so late, but just noticed).